src/Elcodi/Component/Menu/Entity/Menu/Node.php

```diff
@@ -290,6 +297,10 @@ public function isActive($currentUrl)
      */
     public function isExpanded($currentUrl)
     {
+        if (!($this->subnodes instanceof Collection)) {
```

Comment:
Good then :D Thanks for that info! ^^ I was very used of working with these parenthesis :dancer: